openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataInsight.spec.ts

```diff
@@ -145,7 +145,7 @@ describe('Data Insight feature', { tags: 'Observability' }, () => {
 
     // Adding a manual wait to allow some time between deploying the pipeline and triggering it
     // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(1000);
+    cy.wait(2000);
     cy.get('[data-testid="run-now-button"]').click();
```

Comment:
Instead a manual wait can we have timeout for getting run-now button or may be to have that button enabled?

Reply:
As pointed out earlier this is the same place in different spec i just forgot to update here,  after hitting deploy button things are handled by BE, and UI does not have any dependency to enable or disable the button.